samples/cpp/CMakeLists.txt

```diff
@@ -2,13 +2,18 @@
 # SPDX-License-Identifier: Apache-2.0
 #
 
+# cmake_minimum_required (VERSION 3.15)
 cmake_minimum_required (VERSION 3.10)
```

Comment:
do we need to use 3.15 as minimum required for Windows ?

Reply:
Yes. You are right. This is just for test purpose to test the set cxx_flags solution. I will enable both the fix at the end.